san/tests/test_sanbase_integration.py

```diff
@@ -10,12 +10,23 @@
     "project_slug":  "santiment",
     "from_date": month_ago(),
     "to_date": two_days_ago(),
-    "interval": "1d"
+    "interval": "1d",
+    "address": "0x1f3df0b8390bb8e9e322972c5e75583e87608ec2"
 }
 
+METRICS_USING_ETHEREUM = ["gas_used", "miners_balance", "mining_pools_distribution"]
+
+def ordinary_function_maker(query, slug=params['project_slug']):
+    result = san.get(query+'/'+slug,
+        from_date=params['from_date'],
+        to_date=params['to_date'],
+        interval=params['interval']
+        )
+    return result
+
 @attr('integration')
 def test_batched_queries_equal_format():
-    queries = sanbase_graphql.QUERY_MAPPING.keys()
+    queries = sanbase_graphql.QUERY_MAPPING.keys() - METRICS_USING_ETHEREUM
```

Comment:
Why are metrics using ethereum not included?

Reply:
Because I test out every function with equal format, which can use "santiment" as a slug. Then I did another for cycle, so that I cover the ones that use "ethereum" also.